### DISCUSSION TITLE
Debian package?

### DISCUSSION THREAD

**Discussion by rbuelund** [gammu/gammu#549] [General]
Right now it is only possible to install version 1.40 with "apt-get install gammu" on a Debian system. Is there gonne be a prebuild package of version 1.42 for the Debian system ? I tried to compile it myself on a raspberry - but I cannot get it to work :-(

**Comment by nijel** [comment]
1.41 is available in Sid (Debian unstable), see https://packages.debian.org/sid/gammu

The 1.42 is not yet there, the package [is looking for new maintainer](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=951855).
